src/sentry/tasks/reports.py

```diff
@@ -221,7 +221,7 @@ def _query_tsdb_groups_chunked(func, issue_ids, start, stop, rollup):
     return combined
 
 
-def prepare_project_series(start__stop, project):
+def build_project_series(start__stop, project):
```

Comment:
This kind of feels like an arbitrary distinction between "build" and "prepare".

If you want to communicate more specifically that they fetch the data and imply that they perform IO, why not just use "fetch" instead of a less abstract verb?